BranchSDK.podspec

```diff
@@ -18,7 +18,7 @@ Use the Branch SDK (branch.io) to create and power the links that point back to
   s.ios.deployment_target = '12.0'
   s.tvos.deployment_target = '12.0'
 
-  s.resource = "Sources/Resources/PrivacyInfo.xcprivacy"
+  s.resource_bundles = { 'BranchSDK' => 'Sources/Resources/*.xcprivacy' }
```

Comment:
Q: Did the prior `s.resource` assignment replace the privacy manifest entirely? Or was it merged into a bigger resulting privacy manifest?